Jenkinsfile

```diff
@@ -1,6 +1,13 @@
+// Jenkinsfile: scripted Jenkins pipefile
+// [revision: Jenkinsfile@20191122-02]
+// This file was automatically generated using 'python -m vsc.install.ci -f'
+// DO NOT EDIT MANUALLY
+
 node {
-  stage 'checkout git'
-  checkout scm
-  stage 'test'
-  sh "python setup.py test"
-}
+    stage 'checkout git'
+    checkout scm
+    stage 'test'
+    sh 'python2.7 -V'
```

Comment:
what is this for?

Reply:
To be very sure that we're still testing with Python 2.7, since we're configuring `tox` to ignore missing interpreters (so we can get this merged on `jenkins8` where Python 3.6 is not available yet.